style.css

```diff
@@ -0,0 +1,267 @@
+}
+.awesome__subtitle {
+    margin-bottom: 110px;
+}
+.awesome__container {
+    display: flex;
+    justify-content: space-between;
+}
+.awesome__block {
+    width: 370px;
+    text-align: center;
+}
+.awesome__icon-wrapper {
+    width: 92px;
+    height: 92px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    border: 1px solid #EC7979;
+    border-radius: 50%;
+    margin: 0 auto 78px;    
+}
+.awesome__block-title {
+    font-family: 'Open Sans', sans-serif ;
+    margin-bottom:47px;
+}
+/** awesome style end*/
+/** pricing style start*/
+.price {   
+    background: #F5F5F5;
+}
+.price__subtitle {
+    margin-bottom: 79px;
+}
+.price__wrapper {    
+    display: flex;
+    justify-content: space-between;   
+}
+.price__block {
+    width: 370px;
+    height: 510px;
+    text-align: center;  
+    margin-bottom: 50px;  
+}    
+.price__block:hover{
+    filter: drop-shadow(0px 4px 10px rgba(51, 51, 51, 0.2));
+    margin-top: -20px;
+}
+.price__block-name {    
+    height: 56px;display: flex;
+    justify-content: center;
+    align-items: center;
+    font-family: 'Open Sans', sans-serif;
+    font-weight: bold;
+    font-size: 24px;
+    line-height: 33px;
+    text-transform: capitalize;
+    color: #FFFFFF; 
+    background: #333333;
+    border: 2px solid #F5F5F5;      
+    }
+.price__block-cost {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 96px;    
+    font-weight: bold;
+    font-size: 36px;
+    line-height: 42px;    
+    color: #FFFFFF;
+    background: #333333;
+    border: 2px solid #F5F5F5;
+}    
+.price__block-text {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 46px;          
+    color: rgba(255, 255, 255, 0.5);
+    background: #333333;
+    border: 2px solid #F5F5F5;
+}
+.price__list-item {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 46px;      
+    text-transform: capitalize;    
+    color: #86878B;
+    background: #FFFFFF;
+    border: 2px solid #F5F5F5;  
+}
+/** pricing style end*/
+/** customers style start*/
+.customers__subtitle {    
+    margin-bottom: 83px;    
+}
+.customers__container {   
+    display: flex; 
+    flex-direction: row;
+    justify-content: space-between;  
+  }
+.customer__block {
+    margin-bottom: 64px;
+}
+.customer__text {
+    position: relative;
+    display: flex;
+    flex-direction: column;  
+    align-items: center;
+    justify-content: center;
+    width: 370px;
+    height: 180px;      
+    color: #86878B;
+    background: #E2E2E2;
+    border-radius: 10px;
+    margin-bottom: 66px;
+    z-index: 2;                 /*? не перекрывает блок с z-index 1 */
+}
+.triangle {
+    position: absolute;
+    top: 169px;
+    left: 85px;
+    width: 21.2px;
+    height: 21.2px;
+    background: #E2E2E2;
+    transform: rotate(45deg);
+    z-index: 1;                 /*? не получилось */    
```

Comment:
It's not working because elements should be placed near each other and you have one element inside other.